docs/admin/CertificateTransparency.adoc

```diff
@@ -0,0 +1,114 @@
+= Certificate Transparency
+
+== Overview
+
+A basic version of Certificate Transparency (CT) V1 support (rfc 6962) is offered in this release. It has the capability of issuing certificates with embedded Signed Certificate Time stamps (SCTs) from any trusted log where each deployment site chooses to have its root CA cert included. The system can also be configured to support multiple CT logs.  Minimum of one trusted CT log is required for this feature to work.
+
+IMPORTANT: It is the responsibility of the deployment site to establish its trust relationship with a trusted CT log server.
+
+== Configuration
+
+The configuration for CT is in the CA's CS.cfg (/var/lib/pki/<instance name>/ca/conf/CS.cfg).
+
+=== ca.certTransparency.mode
+
+ca.certTransparency.mode specifies CT mode. There are three Certificate Transparency modes:
+
+* *_disabled_*: issued certs will not carry SCT extension
+* *_enabled_*: issued certs will carry SCT extension
+* *_perProfile_*: certs enrolled through those profiles that contain the following policyset will carry SCT extension: SignedCertificateTimestampListExtDefaultImpl
+
+Default is _enabled_
+
+=== ca.certTransparency.log.num
+
+ca.certTransparency.log.num specifies the total number of CT logs defined in the configuration.  Not all defined CT log entries are considered active.  See <<ca.certTransparency.log.<id>.enable>>.
```

Comment:
The link doesn't seem to be working:  `<<ca.certTransparency.log.<id>.enable>>`.
The target probably needs to be a section header, e.g. `==== ca.certTransparency.log.<id>.enable`.

But no worries, we can fix this another time.

Reply:
sure.